### DISCUSSION TITLE
Creating a list of use-cases (that involve personal data sharing)

### DISCUSSION THREAD

**Discussion by anilturkmayali** [International-Data-Spaces-Association/Personal-Data-and-IDS#17] [Ideas]
Creating a list of data sharing patterns (or use-cases, or scenarios) that involve personal data. 

We can use this list for identifying what is needed and verify whether currently existing technologies/building blocks can adequately do the job.

**Comment by anilturkmayali** [comment]
We have also discussed this issue with @AlexTourski and will come up with a suggestion on the matter. 
Any ideas are more than welcome.

**Reply by AlexTourski** [reply]
Thank you, Anil.
Indeed, it is time to go one step deeper into the subject exploration from our perfect declaration which we have just adopted at the last meeting.
And here we need a good language for such exploration. 
We believe that use cases ARE the perfect language to do the job. 
Therefore  I suggest we spent time on it and it will be a great next milestone in our work which will directly lead – at the next level – to the specific path of further technological development. 

Please note that it is difficult to distinguish "Personal Data Space" use cases from "Data Space" use cases, therefore most probably such use cases will go beyond "Personal", yet here let's concentrate on personal use cases first. E.g. we start form the use cases which start with "As a user, I would like...:

To be more specific, the suggestion is: 

1. To split use cases between several "areas" to define the complete scope of the work. I see at least the following areas:
a) Monopoly: "As a user I would like not to depend on monopoly of any platform, (group of) people or organization: e.g. if I publish a post on facebook, I would like to continue on linkedin and so on, and so my friends will do"
b) Data sovereignty. "As a user, I would like to own all my data and control who, how and on what conditions access it via any platform"
c) Data Silos issue: "As a user I would like that any data exists in one – and original – copy, and no other copies exist. All users via ll platforms access only this original copy"
d) IPR control. "As a user, I would like to claim ownership to data and anyone should be able immediately check it up via any platform of her choice"
e) Data Space. "As a user I would like to see all data in the world existing in one copy at source accessible by authorized users via any platforms of their choice. No more government or corporate data bases"
f) Long term preservation. "As a user I would like to be assured that any data I put via any platform will be stored under my control for 500 years and more, without any efforts from my side"
g) No data structure/format/ontology issues. "As a user, I do not want to be troubled with data structure, format or ontology issues. I just want to deal with information in a convenient way"
h) Persistent ID. "As a user, if I have an ID/URL of any asset, I want this asset to be available at this ID/URL 500 years from now. No more broken links or lost data"
i) Long-tail inclusion. "As a user of any capacity, I want to be able to participate in economic, cultural and other interactions at the same level of inclusivity as large corporations or other people"
j) e-reputation. "As a user, I would like to develop and maintain my reputation and my products reputation and I want it to be available and appreciated on my request in any place on the planet"
k) PET (privacy enhancing technologies). "As a user I understand the need for researchers, governments and other actors to get statistics which includes my personal data. I do not mind them to get it as long as my identity could be totally hired on my request". It includes services like e-voting
l) e-money. "As a user, I would like to have ability to transfer/receive money between me and other people or organizations directly, without use of bank accounts"
m) Strong ID management. "As a user, I would like to use one e-passport for all transactions, access to all platforms and services, getting money from ATM, access to rental cars, hotel rooms etc". "As a user I would like to control the level of anonymity during such transactions"
n) Security. "As a user, I want to have a secure environment, where I do not have to worry about any threads, and I am always supported by security services". "As a user I do not want to be a victim of any kind of "cancelling" or other attack"
o) Transparency. "As a user I would like to live in the world where any information is available immediately, e.g. I would like to know how many French-speaking MIT students purchased Colgate toothpaste last year", or "As a user, I would like to know the exact author and e-reputation of every news to avoid fake news".

2. Where needed, to develop a hierarchy of use cases. E.g. the "Strong ID" use case leads to the use case of "controlling the level of anonymity", "revocation", "ID security" etc

3. Every use case to be presented in a structural way, with 
a) clear and concise text of the usecase
b) extended comments
c) applicable environment, limitations etc

4. Where applicable, to split use cases between industrial domains: manufacturing, health, cultural heritage, personal matters etc. 

5. We keep use cases solution-agnostic, so that later on (at the next phase of our work) we always can match certain solutions/ technologies against these use cases

Best Regards,
Alex Tourski
Founder
postplatforms.org
= = = = = = = = = 
mob: +31645388324
email: alex.tourski@postplatforms.org

**Reply by mabuse68** [reply]
I would suggest to look at accountability.
A person has a different accountability than a business entity.
So, there's a plethora of use cases where the person needs to be protected (that's what covers the most attention), but there are a lot of use cases where the business entity interacting with persons needs guarantees!
If I share my personal data and, suddenly, I move it because it's my right to do so, what are the means that a data space participant has to seamlessly access my data, even if hosted somewhere else?
For each action a person undertakes in a regulated environment, these's a cost for the data consumer.

**Reply by AlexTourski** [reply]
Dear Gabriele (sorry, I am not sure whether mabuse68 is you), 
I am not sure I clearly understood your case. 
E.g.
- what does it mean: "needs guarantees"? what kind of guarantees? why? 
- what does it mean:  "...move it"? My personal data is always in one place, presumably. Why should I move it? And, even if I move it, all peers will certainly not notice it (e.g. when I move my web site to a new place you will never know it thanks to DNS). Or do you mean "I changed access rights to certain data to certain group of users"? 
- what does it mean: "regulated environment"? and what are the "cost for the data consumer" you refer to?

Probably it makes sense to re-write the whole case in a "use case language"? Could you make it?
Best Regards,
Alex Tourski

**Comment by AlexTourski** [comment]
Dear All, 
At the last meeting (26 Oct) I promised to come up with the complete picture of the use cases. 
yesterday I made my first attempt. I chose MIRO (mind map). 
As the picture was not very easy to grasp I recorded the [video](https://izi.synology.me:2022/vs/sharing/LX6yQUDK#!bW92aWUtMTk4OQ==
). Please start with it first

Than you may open the [mindmap ](https://miro.com/app/board/uXjVNP7GDzw=/)itself. Pass: 1254a126
